pkgs/tools/compression/lzip/default.nix

```diff
@@ -27,6 +30,11 @@ stdenv.mkDerivation rec {
   doCheck = true;
   enableParallelBuilding = true;
 
+  installPhase = lib.optionalString stdenv.hostPlatform.isMinGW ''
```

Comment:
This will break `lzip` on other platforms, since they now have `installPhase = ""`, right?

Reply:
Oooof. It looks that way. What I wanted was in its not a mingw build, use the default installPhase of builder. Otherwise use the installPhase that I have specified.

I thought if installPhase was empty it would use the default, looking at the rebuild count, that assumption clearly was not true.

Reply:
It would indeed be empty if `installPhase` evaluates to `null`.

Reply:
I think I started by trying to patch the Makefile.in file but ran into issues. Could be mistaken though, its been a little bit. I think what I would like to try and do is fix my logic at the installPhase

Reply:
Fixed.